install_config/configuring_sdn.adoc

```diff
@@ -164,6 +164,181 @@ Check VNIDs by running:
+need to completely take down the cluster temporarily.
+
+. On every node and master, do:
++
+----
+# systemctl stop atomic-openshift-node
+# systemctl restart docker
+----
+
+. As a cluster admin user, run:
++
+----
+$ oc delete hostsubnets --all
+$ oc delete clusternetwork default
+----
+
+. Stop the masters. In a single-master cluster:
++
+----
+# systemctl stop atomic-openshift-master
+----
++
+For multi-master clusters:
++
+----
+# systemctl stop atomic-openshift-master-controllers
+# systemctl stop atomic-openshift-master-api
+----
+
+. Now edit the *_master-config.yaml_* file and make the desired changes.
+
+. Restart the masters:
++
+----
+# systemctl start atomic-openshift-master
+----
++
+or (multi-master):
++
+----
+# systemctl start atomic-openshift-master-controllers
+# systemctl start atomic-openshift-master-api
+----
+
+. Restart the node service on all of the nodes and masters:
++
+----
+# systemctl start atomic-openshift-node
+----
+
+At this point the cluster will be back up, but every
+previously-running pod will have been killed, and it may take a few
+minutes to get everything completely back up and running again.
+
+=== Changing the Service Network IP Range
+
+If you need to change `*serviceNetworkCIDR*`, either because you need
+more than 65,536 service IPs, or because you have belatedly discovered
+a conflict between the service IP range and other IPs on your network,
+you will have to take down the cluster (as above), but you will also
+need to delete and recreate all Service objects. There is no simple
+way to do this, and in particular, to get all of the permissions
+correct, each Service will have to be recreated by the correct user;
+you can't simply have an administrator recreate them. Thus, this will
+result in a more noticeable outage than the `*clusterNetworkCIDR*`
+renumbering case.
+
+. On every node and master, do:
++
+----
+# systemctl stop atomic-openshift-node
+----
+
+. As a cluster admin user, run:
++
+----
+$ oc delete services --all
+$ oc delete endpoints --all
+$ oc delete clusternetwork default
+----
+
+. Stop the masters. In a single-master cluster:
++
+----
+# systemctl stop atomic-openshift-master
+----
++
+For multi-master clusters:
++
+----
+# systemctl stop atomic-openshift-master-controllers
+# systemctl stop atomic-openshift-master-api
+----
+
+. Now edit the *_master-config.yaml_* file and make the desired change
+to `*serviceNetworkCIDR*`.
+
+. Restart the masters:
++
+----
+# systemctl start atomic-openshift-master
+----
++
+or (multi-master):
++
+----
+# systemctl start atomic-openshift-master-controllers
+# systemctl start atomic-openshift-master-api
+----
+
+. Restart the node service on all of the nodes and masters:
++
+----
+# systemctl start atomic-openshift-node
+----
+
+Unlike in the `*clusterNetworkCIDR*`-renumbering case, this will not
+kill previously-running pods. However, it leaves all Services deleted,
+and users will have to recreate them after this.
```

Comment:
And any part of the cluster configuration that relies on the service address, e.g., pulling from the openshift registry  will need to be updated in the DCs and RCs as well. Applications configurations that rely on service ip addresses will need to be changed as well. It can take quite a lot of work to get everything back up and running.